Bukkit/src/main/java/com/plotsquared/bukkit/generator/BukkitPlotGenerator.java

```diff
@@ -159,12 +160,14 @@ public ChunkGenerator getPlatformGenerator() {
             @NonNull BiomeGrid biome
     ) {
 
-        GenChunk result = new GenChunk();
+        int minY = BukkitWorld.getMinWorldHeight(world);
+        int maxY = BukkitWorld.getMaxWorldHeight(world);
+        GenChunk result = new GenChunk(minY, maxY);
```

Comment:
I think the code in `ScopedQueueCoordinator ` (subclass of `GenChunk`) assumes its max is inclusive, so the max here should be inclusive too.